src/Domain/DonationStatus.php

```diff
@@ -5,29 +5,102 @@
 /**
  * @link https://docs.google.com/document/d/11ukX2jOxConiVT3BhzbUKzLfSybG8eie7MX0b0kG89U/edit?usp=sharing
  * @todo Consider vs. Stripe options
```

Comment:
I can't actually access the link - I just moved the comment over from Donation. Should we delete it here?

Not sure what the todo consider means - it's another comment that I moved from the Donation class.

Reply:
@bdsl Yeah, I think we can delete the link as nobody in the current team has access – whatever it was didn't get put in a shared Drive.

I think the note was to think about the pro's and con's of migrating new donations to use Stripe's own status strings verbatim. Given we have use cases where we simplify multiple Stripe statuses to one internal one (e.g. disputes lost are _Refunded_), my view now is that this is unlikely to be worth it. It would buy us slightly more clarity when comparing things to Stripe but with several downsides I can think of off-hand:
* more complexity to manage
* likelihood of regressions while we switch over
* less comparability to historic data from other PSPs
* greater long-term lock-in to using Stripe exclusively